src/Common/Commands.Common/AzurePSCmdlet.cs

```diff
@@ -38,6 +47,63 @@ static AzurePSCmdlet()
             }
 
             AzureSession.ClientFactory.UserAgents.Add(AzurePowerShell.UserAgentValue);
+            InitializeTokenCaches();
+            DefaultProfile = InitializeDefaultProfile();
+            CurrentProfile = DefaultProfile;
+            UpdateSessionStateForProfile(CurrentProfile);
+            AzureSession.DataStore = new DiskDataStore();
```

Comment:
If we are running tests, this will always change the data store to disk one. Instead we should have this line inside this block
```csharp
if (!TestMockSupport.RunningMocked)
{
  ....
}
```